internal/sbi/consumer/nrf_service.go

```diff
@@ -79,59 +84,14 @@ func (s *nnrfService) SendNFStatusNotify(
 		return problemDetails
 	}
 	if res != nil {
-		defer func() {
-			if resCloseErr := res.Body.Close(); resCloseErr != nil {
-				logger.NfmLog.Errorf("NotificationApi response body cannot close: %+v", resCloseErr)
-			}
-		}()
-		if status := res.StatusCode; status != http.StatusNoContent {
-			logger.NfmLog.Warnln("Error status in NotificationPost: ", status)
+		if res.AcceptEncoding != "" && res.AcceptEncoding != "application/json" {
+			logger.NfmLog.Warnln("Unexpected Accept-Encoding in response: ", res.AcceptEncoding)
 			problemDetails := &models.ProblemDetails{
-				Status: int32(status),
+				Status: http.StatusUnsupportedMediaType,
```

Comment:
why is `http.StatusUnsupportedMediaType`?

Reply:
@andy89923 The status is correct. If the server does not accept the `Content-Encoding` sent by the client, the server will send `415 Unsupported Media Type` back to the client. 

@KunLee76 I think the problem is that the openapi client has [already considered this situation](https://github.com/free5gc/openapi/blob/feat/r17/nrf/NFManagement/api_subscriptions_collection.go#L414). So there's no need to do error handling again.